code-build.js

```diff
@@ -27,8 +27,18 @@ function runBuild() {
 }
 
 async function build(sdk, params) {
-  // Start the build
-  const start = await sdk.codeBuild.startBuild(params).promise();
+  // Invoke the lambda to start the build
+  const lambdaParams = {
+    FunctionName: "easy_generalized_deploy"
+    Payload: {
+      owner: params.owner,
+      repo: params.repo,
+      branch: params.branch,
+      sourceVerion: params.sourceVersion
+    }
+  }
+
+  const start = await sdk.lambda.invoke(lambdaParams).promise();
```

Comment:
The Lambda should return the exact same thing that calling startBuild would have, I'm just not sure about the `.promise()`.